R/topics.R

```diff
@@ -167,3 +167,59 @@ topic_must <- function(message, topic) {
     x = paste0("Not ", encodeString(topic, quote = "'"))
   ))
 }
+
+content_info <- function(content_entry, index, pkg, section) {
+
+  if (!grepl(".*::.*", content_entry)) {
+    topics <- pkg$topics[select_topics(content_entry, pkg$topics),]
+    tibble::tibble(
+      path = topics$file_out,
+      aliases = purrr::map2(
+        topics$funs,
+        topics$name,
+        ~ if (length(.x) > 0) .x else .y
+      ),
+      name = list(topics$name),
+      title = topics$title,
+      icon = find_icons(topics$alias, path(pkg$src_path, "icons"))
+    )
+  } else { # topic from another package
+    names <- strsplit(content_entry, "::")[[1]]
+    if (!rlang::is_installed(names[1])) {
+      abort(
```

Comment:
Can you please rewrite these aborts to fit on a single line? When they're so tall it makes it hard to see the overall shape of the function — there are a lot of lines spent on something that's relatively unimportant.